px-map.html

```diff
@@ -12,6 +12,10 @@
 <link rel="import" href="css/px-map-leaflet-styles.html">
 <link rel="import" href="css/px-map-styles.html">
 
+<!-- Import px-icon-set icons -->
+<link rel="import" href="../px-icon-set/px-icon-set.html"/>
```

Comment:
Any way this can be moved further "down" where the icons is actually used/inserted? not sure how this works